### ISSUE TITLE
Deprecated ref as string throw an error

### ISSUE THREAD

**Issue by viktor-equinix** [react-component/dialog#83]
`rc-dialog` component is using React 16, but React 16 considers `ref` string as legacy and in my app it throw an error in [ANT Design component](https://github.com/ant-design/ant-design/blob/master/components/modal/Modal.tsx) which is using `rc-dialog`.
I think the issue is on this line:
https://github.com/react-component/dialog/blob/master/src/Dialog.tsx#L216

common text of error:
```
Uncaught Error: Element ref was specified as a string (header) but no owner was set. This could happen for one of the following reasons:
1. You may be adding a ref to a functional component
2. You may be adding a ref to a component that was not created inside a component's render method
3. You have multiple copies of React loaded
...
```

stack trace:
```
The above error occurred in the <div> component:
    in div
    in div
    in LazyRenderBox
    in AnimateChild
    in Animate
    in div
    in div
    in Dialog
```
Please fix it.

I am using react@16.3.1

Looks like problem appears due (4) https://news.ycombinator.com/edit?id=12093234

> String refs force React to keep track of currently executing component. This is problematic 
> because it makes `react` module stateful, and thus causes weird errors when `react` module is 
> duplicated in the bundle.

**Comment by yesmeck** [comment]
@viktor-equinix Could you open a PR?

**Comment by otakustay** [comment]
We are encountering the same issue, I can make a PR, however is the `body` ref actually used? I can't see any `refs.body` code in `Dialog.tsx`

If it is not used in any case, I may create a PR to simply delete this ref

**Comment by yesmeck** [comment]
It seems the `body` ref is not used anywhere.

**Comment by yesmeck** [comment]
Fixed in 7.1.4
